components/browser/menu/src/main/res/values/dimens.xml

```diff
@@ -11,8 +11,8 @@
     <!--Menu Item -->
     <dimen name="mozac_browser_menu_item_text_size">16sp</dimen>
     <dimen name="mozac_browser_menu_item_container_layout_height">48dp</dimen>
-    <dimen name="mozac_browser_menu_item_container_padding_start">24dp</dimen>
```

Comment:
Is it there any specific reason why do we want to change padding for all the menu items?

Reply:
@Amejia481 https://github.com/mozilla-mobile/fenix/issues/7157

Reply:
Thanks, I just saw the change in isolation web ext!